client/src/pages/Login/Login.tsx

```diff
@@ -51,9 +53,11 @@ export default function Login(): JSX.Element {
               </Grid>
             </Grid>
             <LoginForm handleSubmit={handleSubmit} />
+            <AuthDemo />
           </Box>
-          <Box p={1} alignSelf="center" />
         </Box>
+        <Divider />
+        <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create" />
```

Comment:
This is appearing underneath the bottom of my screen, so I would have to scroll to find it. Can you bring it up a bit?